packages/tailwindcss-language-server/src/projects.ts

```diff
@@ -755,6 +755,7 @@ export async function createProjectService(
         originalConfig = { theme: {} }
       } catch {
         // TODO: Fall back to built-in v4 stuff
+        // TODO: Log errors? It might get noisy as people are editing their CSS though
```

Comment:
You could do something like we do in v3 where you can scope an error to a specific key and only show an error once for that key for the life time of the running process.

Maybe not worth it, just a thought.

Reply:
Yeah I don't think that'd help here because different types of errors might still want to get logged. Even potentially different error messages.